<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TotalInterface</class>
 <widget class="QWidget" name="TotalInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Start">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>开始谈判</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Quit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>退出系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="Description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>625</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>说明：用户直接点击“开始谈判”按钮后显示谈判过程和谈判结果，企业的期望值和可放松值可在右下侧进行自主输入修改，之后点击“开始谈判”按钮即可进行新的谈判。</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="Negotiation_txt">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>124</y>
     <width>311</width>
     <height>280</height>
    </rect>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Scene">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>谈判情景</string>
   </property>
  </widget>
  <widget class="QLabel" name="Data">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>谈判数据录入</string>
   </property>
  </widget>
  <widget class="QLabel" name="A">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>企业A</string>
   </property>
  </widget>
  <widget class="QLabel" name="Profit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>30</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>利润</string>
   </property>
  </widget>
  <widget class="QLabel" name="A_priority">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>优先级</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Priority_A">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="A_expect">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>期望值</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Expect_A">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="A_relaxValue">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>可放松值</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RelaxValue_A">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="B">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>企业B</string>
   </property>
  </widget>
  <widget class="QLabel" name="UnitPrice">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>购买单价</string>
   </property>
  </widget>
  <widget class="QLabel" name="B_priority">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>优先级</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Priority_B">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="B_expect">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>170</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>期望值</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Expect_B">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="B_relaxValue">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>可放松值</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RelaxValue_B">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>200</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="C">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>企业C</string>
   </property>
  </widget>
  <widget class="QLabel" name="TotalCost">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>原料固定总费用</string>
   </property>
  </widget>
  <widget class="QLabel" name="UnitCost">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>原料变动单位费用</string>
   </property>
  </widget>
  <widget class="QLabel" name="C_priority">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>优先级</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Priority_C_total">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Priority_C_unit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>300</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="C_expect">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>240</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>期望值</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Expect_C_total">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Expect_C_unit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>300</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="C_relaxValue">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>240</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>可放松值</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RelaxValue_C_total">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="RelaxValue_C_unit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>300</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="D">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>企业D</string>
   </property>
  </widget>
  <widget class="QLabel" name="ShipFee">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>370</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>运输费</string>
   </property>
  </widget>
  <widget class="QLabel" name="D_priority">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>优先级</string>
   </property>
  </widget>
  <widget class="QLabel" name="D_expect">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>期望值</string>
   </property>
  </widget>
  <widget class="QLabel" name="D_relaxValue">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>340</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>可放松值</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Priority_D">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>370</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="Expect_D">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>370</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="RelaxValue_D">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>370</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <zorder>Start</zorder>
  <zorder>Quit</zorder>
  <zorder>Description</zorder>
  <zorder>Negotiation_txt</zorder>
  <zorder>Scene</zorder>
  <zorder>Data</zorder>
  <zorder>A_relaxValue</zorder>
  <zorder>A</zorder>
  <zorder>Profit</zorder>
  <zorder>A_priority</zorder>
  <zorder>Priority_A</zorder>
  <zorder>A_expect</zorder>
  <zorder>Expect_A</zorder>
  <zorder>RelaxValue_A</zorder>
  <zorder>B</zorder>
  <zorder>UnitPrice</zorder>
  <zorder>B_priority</zorder>
  <zorder>Priority_B</zorder>
  <zorder>B_expect</zorder>
  <zorder>Expect_B</zorder>
  <zorder>B_relaxValue</zorder>
  <zorder>RelaxValue_B</zorder>
  <zorder>C</zorder>
  <zorder>TotalCost</zorder>
  <zorder>UnitCost</zorder>
  <zorder>C_priority</zorder>
  <zorder>Priority_C_total</zorder>
  <zorder>Priority_C_unit</zorder>
  <zorder>C_expect</zorder>
  <zorder>Expect_C_total</zorder>
  <zorder>Expect_C_unit</zorder>
  <zorder>C_relaxValue</zorder>
  <zorder>RelaxValue_C_total</zorder>
  <zorder>RelaxValue_C_unit</zorder>
  <zorder>D</zorder>
  <zorder>ShipFee</zorder>
  <zorder>D_priority</zorder>
  <zorder>D_expect</zorder>
  <zorder>D_relaxValue</zorder>
  <zorder>Priority_D</zorder>
  <zorder>Expect_D</zorder>
  <zorder>RelaxValue_D</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
